src/qibolab/instruments/qblox/controller.py

```diff
@@ -16,6 +16,7 @@
 MAX_BATCH_SIZE = 30
 """Maximum number of sequences that can be unrolled in a single one (independent of measurements)."""
 SEQUENCER_MEMORY = 2**17
+REFERENCE_CLOCK = "internal"
```

Comment:
The enum in `cluster.py` was actually fine, you could migrate it here.

Reply:
Why we need an enum classes if this choice is only between two parameter (internal and external) and we always use the same (as long as the lab configuration doesn't change)?
For sure is better to move this in the create method, so that the user is able to change this reference clock.

Reply:
About *always using the same*, I don't know, you should ask the lab people if this will always be the case. Of course, this could depend on specific lab choices, but there are not many labs using Qibolab and Qblox. So, if that's the case, the constant is even fine.

Instead, if the choice could be different, then the idea is that you want to mirror the possible options with a type.
If you use a `str`, it has infinitely many values (the set of `str` -> just limited by the memory available). But the enum only had two values, the only possible ones.

When the values are two, there is always an alternative: use booleans. Since types in general have no internal algebra (they are just sets), all sets with the same cardinality are isomorphic, so `bool` is naturally mapped to any other type with two values.

Reply:
Ok thanks for the explanation @AleCandido, do you think something like this can work?
```python
    def __init__(self, name, address: str, modules, internal_reference_clock: bool = True):
        """Initialises the controller."""
        super().__init__(name=name, address=address)
        self._reference_clock = "internal" if internal_reference_clock else "external"
        ...
```

Otherwise if you don't like this solution I will restore the enum.

Reply:
Indeed, that's exactly what I meant with the `bool` alternative :)